### PR TITLE
Add term: Javascript Objects - isExtensible

### PR DESCRIPTION
**Pull request by Jamesllllllllll**
### Description

Added term entry for Javascript Objects: `isExtensible()`

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/objects/terms/isExtensible/isExtensible.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+Title: '.isExtensible()'
+Description: 'Returns a boolean value depending on whether a given object can have properties added to it.'
+Subjects:
+  - 'Web Development'
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/objects/terms/isExtensible/isExtensible.md:5]:
```suggestion
  - 'Web Development'
  - 'Code Foundation'
```

File: content/javascript/concepts/objects/terms/isExtensible/isExtensible.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+Tags:
+  - 'Methods'
+  - 'Objects'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **`.isExtensible()`** static method returns `true` if the given object can have properties added to it and `false` if the object cannot have properties added to it.
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/objects/terms/isExtensible/isExtensible.md:14]:
```suggestion
The **`.isExtensible()`** is a static method that returns `true` if the given object can have properties added to it and `false` if the object cannot have properties added to it.
```

File: content/javascript/concepts/objects/terms/isExtensible/isExtensible.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+The output would be:
+
+```shell
+true
+false
+```
+
+## Codebyte Example
+
+Following is a runnable example that demonstrates the `Object.isExtensible()` static method.
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/objects/terms/isExtensible/isExtensible.md:51]:
```suggestion
Following is a runnable example that demonstrates the `Object.isExtensible()` static method:
```

File: content/javascript/concepts/objects/terms/isExtensible/isExtensible.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+```pseudo
+Object.isExtensible(newObject);
+```
+
+Objects are extensible by default. An object can be made non-extensible by using `Object.freeze()`, `Object.preventExtensions`, ot `Object.seal()`. A non-extensible object can still have its properties modified.
+
+## Example
+
+Checking if an object is extensible.
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/objects/terms/isExtensible/isExtensible.md:26]:
```suggestion
Checking if an object is extensible using the `.isExtensible()` method:
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
I have few changes from side :)

File: content/javascript/concepts/objects/terms/isExtensible/isExtensible.md (hunk 1 above)
Comment (Jamesllllllllll) [reply content/javascript/concepts/objects/terms/isExtensible/isExtensible.md:5]: Thanks! I made a slight adjustment to your suggestion, 'Code Foundations' with an "s" [from the list here](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md).

**Comment by dakshdeepHERE** [comment]
Thank you @Jamesllllllllll for the commits I will move this forward for second review. 🚀

File: content/javascript/concepts/objects/terms/isExtensible/isExtensible.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
+---
+Title: '.isExtensible()'
+Description: 'Returns a boolean value depending on whether a given object can have properties added to it.'
+Subjects:
+  - 'Web Development'
+  - 'Code Foundations'
+
```
Comment (avdhoottt) [inline content/javascript/concepts/objects/terms/isExtensible/isExtensible.md:7]:
```suggestion
```

File: content/javascript/concepts/objects/terms/isExtensible/isExtensible.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+---
+
+The **`.isExtensible()`** is a static method that returns `true` if the given object can have properties added to it and `false` if the object cannot have properties added to it.
+
+## Syntax
+
+```pseudo
+Object.isExtensible(newObject);
+```
+
```
Comment (avdhoottt) [inline content/javascript/concepts/objects/terms/isExtensible/isExtensible.md:23]: Add a line describing the `newObject` parameter after the syntax block (Remember to give a line space between them).

File: content/javascript/concepts/objects/terms/isExtensible/isExtensible.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+Objects are extensible by default. An object can be made non-extensible by using `Object.freeze()`, `Object.preventExtensions`, ot `Object.seal()`. A non-extensible object can still have its properties modified.
```
Comment (avdhoottt) [inline content/javascript/concepts/objects/terms/isExtensible/isExtensible.md:24]: Add this as a note, take reference on how to add a note from Codecademy's [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md).

File: content/javascript/concepts/objects/terms/isExtensible/isExtensible.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+};
+
+console.log(Object.isExtensible(characters));
+
+Object.preventExtensions(characters);
+
+console.log(Object.isExtensible(characters));
+```
+
+The output would be:
```
Comment (avdhoottt) [inline content/javascript/concepts/objects/terms/isExtensible/isExtensible.md:44]:
```suggestion
The output for the above will be:
```
Try to be specific and sure as possible.

File: content/javascript/concepts/objects/terms/isExtensible/isExtensible.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+
+```pseudo
+Object.isExtensible(newObject);
+```
+
+Objects are extensible by default. An object can be made non-extensible by using `Object.freeze()`, `Object.preventExtensions`, ot `Object.seal()`. A non-extensible object can still have its properties modified.
+
+## Example
+
+Checking if an object is extensible using the `.isExtensible()` method:
```
Comment (avdhoottt) [inline content/javascript/concepts/objects/terms/isExtensible/isExtensible.md:28]:
```suggestion
The following code will check if an object is extensible or not using the `.isExtensible()` method:
```

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@Jamesllllllllll Few changes from my side!!

**Comment by Jamesllllllllll** [comment]
Thanks @avdhoottt - I made the requested changes.

File: content/javascript/concepts/objects/terms/isExtensible/isExtensible.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+
+```js
+const characters = {
+  ArthurDent: 'Earth',
+  FordPrefect: 'Betelgeuse Five',
+  ZaphodBeeblebrox: 'Betelgeuse Seven',
+};
+
+console.log(Object.isExtensible(characters));
+
```
Comment (avdhoottt) [inline content/javascript/concepts/objects/terms/isExtensible/isExtensible.md:39]:
```suggestion
```

File: content/javascript/concepts/objects/terms/isExtensible/isExtensible.md [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+```js
+const characters = {
+  ArthurDent: 'Earth',
+  FordPrefect: 'Betelgeuse Five',
+  ZaphodBeeblebrox: 'Betelgeuse Seven',
+};
+
+console.log(Object.isExtensible(characters));
+Object.preventExtensions(characters);
+
```
Comment (avdhoottt) [inline content/javascript/concepts/objects/terms/isExtensible/isExtensible.md:40]:
```suggestion
```

File: content/javascript/concepts/objects/terms/isExtensible/isExtensible.md [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+## Syntax
+
+```pseudo
+Object.isExtensible(newObject);
+```
+
+- `newObject`: It is the object to be checked whether it is extensible.
+
+> **Note:** Objects are extensible by default. An object can be made non-extensible by using `Object.freeze()`, `Object.preventExtensions`, ot `Object.seal()`. A non-extensible object can still have its properties modified.
```
Comment (avdhoottt) [inline content/javascript/concepts/objects/terms/isExtensible/isExtensible.md:25]:
```suggestion
> **Note:** Objects are extensible by default. An object can be made non-extensible by using `Object.freeze()`, `Object.preventExtensions`, or `Object.seal()`. A non-extensible object can still have its properties modified.
```

**Review by avdhoottt — APPROVED** [review]
Looks good to get merged.

**Review by dakshdeepHERE — APPROVED** [review]

**Pull request merged**